File: Data Structures & Algorithms/majority-element/submission-1.java
class Solution {
    public int majorityElement(int[] nums) {
        HashMap<Integer, Integer> majMap= new HashMap<>();

        for (int i = 0 ; i < nums.length ; i++){
            majMap.put(nums[i],majMap.getOrDefault(nums[i], 0) + 1);
        }

        for (Map.Entry<Integer, Integer> entry : majMap.entrySet()){
            if (entry.getValue() > Math.floor(nums.length / 2)){
                return entry.getKey();
            }
        }
        return -1;
    }
}

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-1.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        k = 0
        i = 1

        while i < len(nums):
            if nums[i] == nums[k]:
                # nums[i] = 0
                i += 1
            else:
                k += 1
                nums[k] = nums[i]
                # nums[i] = 0
                i += 1
        
        return k + 1